# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tlp/fix-sh.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

The shell scripts do not fully work with any (e.g. mksh) Shell, so use Bash.

--- TLP-1.8.0/Makefile.vanilla	2025-03-15 23:50:45.719268693 +0200
+++ TLP-1.8.0/Makefile	2025-03-15 23:51:28.863939439 +0200
@@ -141,6 +141,7 @@
 	rm -f bat.d/*~
 
 install-tlp: all
+	sed -i 's,^#!/bin/sh,#!/bin/bash,' $(SHFILES)
 	# Package tlp
 	install -D -m 755 tlp $(_SBIN)/tlp
 	install -D -m 755 tlp-rf $(_BIN)/bluetooth
